### DISCUSSION TITLE
Can i show horizontal Scroll bar for a Detailed list?

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3918] [Q&A]
I have added a Search Result, with those settings: -

![image](https://github.com/user-attachments/assets/3a63a1b8-55ba-4cfc-9be5-a96ae3d687f3)

And i added those slots which are linked to Search managed properties:-

![image](https://github.com/user-attachments/assets/83cd21ef-f39f-4d4f-891c-c8e67f4c9cf8)

and i added those to the Detailed list where all fields are been set a resizable, as follow:-

![image](https://github.com/user-attachments/assets/68d180cc-622a-40b0-87c9-97fc8e5a4918)

![image](https://github.com/user-attachments/assets/8cb5cd43-f69f-42bb-858e-48d94b373012)

now i want to add around 15 columns to the detailed list, but seems i will not get any horizontal Scroll bar by default on top and on the end of the list, to be able to view all the properties... any way to fix this?

Also the columns are not resizable? i cannot expand their width.

I also tried to set the columns as not multiline, as follow:-

![image](https://github.com/user-attachments/assets/04e1443c-02ba-4b25-9a00-596478828a19)

but still i can not expand the width of the columns, here are my display settings:-

![image](https://github.com/user-attachments/assets/e08375de-6705-4fe6-94f3-e36037ebb88b)

Any advice? on how to make the columns resizable + how to show a horizontal tool bar at the top and the button of the list view to allow users to view all the columns for the items?

**Comment by wobba** [comment]
From the component we use there is no way to have user adjustable column width. You can play with a custom template and any html/css you have to accomplish what you want in terms of scrolling.
